src/components/inputs/content-switcher/ohri-content-switcher.component.tsx

```diff
@@ -20,6 +21,15 @@ export const OHRIContentSwitcher: React.FC<OHRIFormFieldProps> = ({ question, on
     }
   }, [question]);
 
+  useEffect(() => {
+    if (!isEmpty(previousValue)) {
+      const { value } = previousValue;
```

Comment:
Due to the differences in datatypes of the `previousValues` for specific controls, otherwise I think the format is pretty standard across the board, maybe expound a bit?

Reply:
This isn't a show stopper. There other controls that access it with `previousValue.value` and others do is this way based on your PR